pkg/ui/react-app/src/pages/graph/Panel.tsx

```diff
@@ -296,6 +313,23 @@ class Panel extends Component<PanelProps & PathPrefixProps, PanelState> {
             </Checkbox>
           </Col>
         </Row>
+        {stores.length > 0 ? (<Row>
```

Comment:
Cases like this are called conditional rendering in React and it's better to use `&&` instead of the ternary operator.

So instead of `{condition ? <Component /> : ""}` we can just do `{condition && <Component />}`

Also, if `stores` is null or undefined here, it'll cause and error too, you can do `{stores?.length > 0 ....` to prevent this.